Plugins~/Src/MeshSyncClientBlender/python/unity_mesh_sync_common.py

```diff
@@ -66,6 +79,54 @@ def msb_on_animation_settings_updated(self = None, context = None):
     # nothing to do for now
     return None
 
+def msb_on_unity_project_path_updated(self = None, context = None):
+    #TODO invoke callback for installing meshsync
+    return None
+
+def msb_get_editor_path_prefix(hub_path = None):
+    prefix = hub_path
+    if prefix is None:
```

Comment:
> the location of the hub itself can also be configured

This is configurable in the Unity Project panel:
![image](https://user-images.githubusercontent.com/51912249/189225048-4e659d71-103a-4d18-a460-47ac87ec348b.png)

> the actual location of the editors can be configured in the hub
This will need to be tackled. A better design might be to have a DIR property for the Editors folder instead of the Hub

Reply:
Changed to point to the editors folder.